crates/solidity/testing/snapshots/bindings_output/yul/try_params/input.sol

```diff
@@ -0,0 +1,13 @@
+interface Service { function test() external returns (uint); }
```

Comment:
+1 on test focus. I suggest removing things like `Service` and possibly `add()`. This asserts only the intended functionality, and reduces noise in reviews. e.g.:

```solidity
try true returns (uint v) {
    assembly {
        v := 0
    }
} catch {
}
```